src/box/lua/config/applier/metrics.lua

```diff
@@ -0,0 +1,30 @@
+local log = require('internal.config.utils.log')
+
+local function apply(config)
+    local configdata = config._configdata
+    local include = configdata:get('metrics.include', {use_default = true})
+    local exclude = configdata:get('metrics.exclude', {use_default = true})
+    local labels = configdata:get('metrics.labels', {use_default = true})
+
+    local default_include = { 'all' }
+    local default_exclude = { }
+    local default_labels = { alias = configdata._instance_name }
+
+    include = include or default_include
+    exclude = exclude or default_exclude
+    labels = labels or default_labels
+
+    box.cfg.metrics.include = include
+    log.verbose('metrics.apply: include applied')
+
+    box.cfg.metrics.exclude = exclude
+    log.verbose('metrics.apply: exclude applied')
+
+    box.cfg.metrics.labels = labels
+    log.verbose('metrics.apply: labels applied')
```

Comment:
Even if it works, it is definitely not a proper way to set a `box.cfg()` parameter. Let's pass all the parameters into `box.cfg()` in the `box_cfg` applier.